platform/core/src/runs/TransactingMutationListener.js

```diff
@@ -38,9 +44,28 @@ define([], function () {
                     transactionService.startTransaction();
                 }
 
+                if (!MUTATION_TRACKER.has(domainObject)) {
+                    MUTATION_TRACKER.set(domainObject, domainObject
+                        .getCapability('mutation')
+                        .listen(function () {})
+                    );
+                }
+
+                cacheService.put(domainObject.getId(), domainObject.getModel());
+
+                function unlistenAndCall(f) {
+                    return function () {
+                        if (MUTATION_TRACKER.has(domainObject)) {
```

Comment:
I'd expect this to always return true, because if a reference to this anonymous function is ineligible for GC, this `domainObject` will also be ineligible for GC by way of the closure